src/expdata_config_sample.yaml

```diff
@@ -0,0 +1,66 @@
+# Configuration parameters to download experimental data and/or edit its format,
+# used by the script get_expdata.py.
+#
+# You can specify one or more blocks for HEPData, PHENIX, or EditData or any combination,
+# and they will all be processed according to their appropriate scripts, steered by get_expdata.py.
+
+# HEPData
+# Any top-level entry with "hepdata" in the name will be processed by get_hepdata.py
+# Each variable consists of a list with one entry per dataset (exceptions: filepath,data_vrs,data_str)
+# You can provide all hepdata entries in a list (as in "hepdata" block), or you can
+# provide separate config entries for one or more hepdata entries (as in "hepdata_CMS_HadronRAA_010" block).
+#-----------------------------------------------------------------------------------------------
+hepdata:
+  filepath: '../input/MATTERLBT1-test/'
```

Comment:
After some thoughts, perhaps it's easier for users if we run the script at the base level (instead of going into the src folder).  So we could remove the "../" in the beginning in the example.

Reply:
Sure, no preference from my side